project.clj

```diff
@@ -13,41 +13,41 @@
   :java-source-paths ["java"]
 
   :dependencies [[org.clojure/clojure "1.8.0"]
-                 [io.sarnowski/swagger1st "0.21.0"]
+                 [org.zalando/swagger1st "0.22.0-beta2" :exclusions [org.apache.httpcomponents/httpcore]]
                  [org.zalando.stups/txdemarcator "0.7.0"]
                  [com.stuartsierra/component "0.3.1"]
-                 [digest "1.4.4"]
-                 [ring "1.4.0"]
+                 [digest "1.4.5"]
+                 [ring "1.5.0"]
                  [org.eclipse.jetty/jetty-servlet "9.2.10.v20150310"]  ; needs to be in sync with ring dependency
```

Comment:
I decided to go with exactly the same version that ring comes with. Did not
try to upgrade.

Am 07.09.2016 8:43 vorm. schrieb "Nikolaus Piccolotto" <
notifications@github.com>:

> In project.clj
> <https://github.com/zalando/friboo/pull/81#discussion_r77767274>:
>
> >                   [org.zalando.stups/txdemarcator "0.7.0"]
> >                   [com.stuartsierra/component "0.3.1"]
> > -                 [digest "1.4.4"]
> > -                 [ring "1.4.0"]
> > +                 [digest "1.4.5"]
> > +                 [ring "1.5.0"]
> >                   [org.eclipse.jetty/jetty-servlet "9.2.10.v20150310"]  ; needs to be in sync with ring dependency
>
> So you updated ring, but not this one. Did you try? Looks like the latest
> stable is 9.3.11.v20160721
> <https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-servlets/9.3.11.v20160721>
> .
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zalando/friboo/pull/81/files/e074a55726717c32700bc546c28265ccd09f72af#r77767274>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGm2hBA3cikuR9uYLip0kedwjqOwplgxks5qnl0CgaJpZM4J2jUz>
> .
>